WORKING_GROUPS.md

```diff
@@ -12,61 +12,19 @@ Groups are not formed to *begin* a set of tasks but instead are formed
 once that work is already underway and the contributors
 think it would benefit from being done as an autonomous project.
 
-If the work defined in a Working Group charter is completed, the Working Group
-should be dissolved and the responsibility for governance absorbed back into
-the TSC. A Working Group can be dissolved either through consensus of the
-Working Group membership or normal TSC motion and vote.
+If the work defined in a Working Group's charter is complete, the charter
+should be revoked.
 
-## Process:
+A Working Group's charter can be revoked either by consensus of the Working
```

Comment:
Nit: I wonder if `consent` might be better than `consensus`. I'm OK either way, though.